### PR TITLE
styling/event details component

### PR DESCRIPTION
**Pull request by cgolinski**
In this PR:
- Styled event details component
- Accessed start and end time

To note:
- Map styles including need to modified within _map.scss not _event-details.scss
- eventDate consts in EventDetails.js are essentially a duplicate of the logic in Event.js component. To keep our code DRY, this logic should be extracted into a helper function in a future PR.
- Very open to design change in a future PR. Not at all attached to this styling.
- Event description is very long for most events. Not sure how to deal with this. Perhaps there's a way to cap the number of characters, and then link to EventBrite for more detail?

### CODE REVIEW

File: src/redux/reducers/event.reducer.js [hunk 1]
```diff
@@ -15,7 +16,13 @@ export const eventReducer = (state = DEFAULT_STATE, action) => {
       return {...state, event: action.payload, isFetching: false, didInvalidate: false}
     case ACTION_TYPES.SET_EVENT_INVALIDATE:
       return {...state, isFetching: false, didInvalidate: true, event: null}
+    case ACTION_TYPES.UPDATE_EVENT_DETAIL_RENDER:
+      console.log('hi');
```
Comment (turquoisemelon) [inline src/redux/reducers/event.reducer.js]: Is it possible to remove console.log from here?

**Comment by turquoisemelon** [comment]
Left a comment. Looks good otherwise. Great job 👍

**Comment by auredufour** [comment]
👍

**Comment by epq** [comment]
Looks good for the most part. 
I noticed a a bug related to the event preview and how it is stored. I'm good with you merging this for our demo and figuring this out later, though!

Initially, state.event.isPreview = true. When I go to an event and click "show more", state.event.isPreview = false. state.event.isPreview stays false, so no there are no previews shown for any other event.

File: src/redux/reducers/event.reducer.js (hunk 1 above)
Comment (cgolinski) [inline src/redux/reducers/event.reducer.js]: Haha, whoops. Done.

**Comment by cgolinski** [comment]
@epq Good catch! We can fix this by resetting eventIsPreview to true on locationChange. I'm going to leave it for the next PR though, so we get this merged before the demo.

File: src/styles/App/_event-details.scss [hunk 2]
```diff
@@ -16,41 +16,84 @@
...
-  padding: 0 0 0 60px; 
+.event-details--time {
   margin: 0;
+  padding-bottom: 10px;
 }
 
-.event-details--description-details {
+.event-details-container-description {
   text-align: left;
   padding: 20px 60px;
```
Comment (cgolinski) [inline src/styles/App/_event-details.scss]: Done. Thanks for the suggestion!

**Pull request merged**